perspective_reddit_bot/check_mod_actions.py

```diff
@@ -0,0 +1,110 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""A reddit bot to detect which actions subreddit moderators actually took."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import argparse
+from datetime import datetime
+import json
+import praw
+import time
+
+from creds import creds
+
+
+def write_moderator_actions(reddit,
+                            line,
+                            id_key,
```

Comment:
Optional/nit: I understand the name, but to me something like `id_column_name` or `id_col_name` or `id_col` seems more natural / is what I'd expect.

Alternatively, instead of passing in the line with the 2 keys, we could do the parsing outside this function and just pass in the comment_id and the timestamp_string directly.

Reply:
Leaving it for now. I didn't feel right changing this to key as I'd have to change the flag for consistency and then the help line didn't feel intuitive. I may revisit the refactor later but it has pros/cons.